schema/metadata_cloud.json

```diff
@@ -15,10 +15,11 @@
       ]
     },
     "device_key": {
-      "type": "boolean"
+      "type": "boolean",
+      "description": "Whether the device authenticates via a private key. Typically false for devices which are proxied for by an IoT core gateway"
     },
     "is_gateway": {
-      "description": "If the device functions as an IoT Gateway, proxying for other devices using a single logical connection",
+      "description": "If the device functions as an IoT Core Gateway, proxying for other devices using a single logical connection",
```

Comment:
"Core" is overly specific here -- at this part of UDMI it's not specific to IoT Core (GCP), but it is about an IoT Gateway. This also still applies, e.g., to AWS etc...